src/pages/404.tsx

```diff
@@ -22,17 +26,23 @@ export default function Custom404() {
 
           <h3>
             That page doesn’t exist (or never did). Let’s get you back where you
-            belong.
+            belong, or send us a dispatch to notify us of an error in the
+            system.
           </h3>
-
-          <LinkTo className="usa-button usa-button--secondary" href="/">
-            Take me home
-          </LinkTo>
+          <div className="flex-align-center">
+            <LinkTo className="usa-button usa-button--secondary" href="/">
+              Take me home
+            </LinkTo>
+            <a
+              href={`mailto:${FEEDBACK_EMAIL}?subject=${FEEDBACK_SUBJECT}`}
+              target="_blank"
+              rel="noreferrer noopener"
+              className="usa-button usa-button--outline usa-button--inverse">
```

Comment:
Question: These links don't have a `hover` state, is that on purpose?

Reply:
no it is not

Reply:
they do have a hover state on my end? the outline button goes from cosmic-latte-light to white, and the secondary button gets a dark background with white text